tools/svg-to-font.js

```diff
@@ -35,7 +37,12 @@ const writeCodepoints = () => {
 }
 
 const svg2woff2 = async (fontBuffer) => {
-	const ttf = svg2ttf(fontBuffer.toString(), {});
+	// Grab timestamp of last commit that affected the source SVG folder. TTFs
+	// store timestamp as per spec, and svg2ttf has an override we can use to
+	// prevent generating a slightly different font file on each run.
+	const last_svg_commit_timestamp = execSync('git log -1 --first-parent --format=%ct ' + srcSvgDir).toString().trim();
```

Comment:
I'd recommend using spawnSync instead. As long as you don't pass it `{ shell: true }` it avoids having to worry about escaping the args.
```suggestion
	const last_svg_commit_timestamp = spawnSync( 'git', [ 'log', '-1', '--first-parent', '--format=%ct', srcSvgDir ] ).toString().trim();
```